apps/challenges/views.py

```diff
@@ -2,6 +2,7 @@
 import json
 import logging
 import os
+from django.shortcuts import get_object_or_404
```

Comment:
I don't think we use this anywhere except home page. Can you please fix this? 
Use a regular error instead of this. Things should be consistent.